packages/wonder-blocks-dropdown/components/search-text-input.js

```diff
@@ -60,6 +70,13 @@ export default class SearchTextInput extends React.Component<Props, State> {
         );
     }
 
+    static defaultProps = {
+        labels: {
+            clearSearch: defaultLabels.clearSearch,
+            filter: defaultLabels.filter,
+        },
+    };
```

Comment:
I notice you're not doing this `this.labels` here, any reason?

Reply:
Yes, this component is only accessed by `MultiSelect` and we are doing the corresponding `this.labels` check in that component. In this case, we have two options:

- No custom labels set: It will use the values from `defaultValues`.
- Custom labels set: It will override the values of `this.props.labels` with the values passed from `MultiSelect`.

Reply:
Disregard my previous comment. We still need to sync the `labels` sent by the parent component, so I might to reuse the same logic to get the current `labels` from `MultiSelect`.